1.7/administration/security-and-authentication/configure-your-security.md

```diff
@@ -0,0 +1,44 @@
+---
+post_title: Configuring Your Security
+menu_order: 1
+---
+
+## Using your own certificate for Admin Router
```

Comment:
Can we get a link to letsencrypt here? I'd like to show how to get a real certificate.